src/mathediting.js

```diff
@@ -52,17 +52,19 @@ export default class MathEditing extends Plugin {
 	_defineSchema() {
 		const schema = this.editor.model.schema;
 		schema.register( 'mathtex-inline', {
+			allowIn: 'span',
 			allowWhere: '$text',
 			isInline: true,
 			isObject: true,
-			allowAttributes: [ 'equation', 'type', 'display' ]
+			allowAttributes: [ 'equation', 'type', 'display', 'fontSize' ]
 		} );
 
 		schema.register( 'mathtex-display', {
+			allowIn: 'span',
 			allowWhere: '$block',
 			isInline: false,
 			isObject: true,
-			allowAttributes: [ 'equation', 'type', 'display' ]
+			allowAttributes: [ 'equation', 'type', 'display', 'fontSize' ]
 		} );
```

Comment:
I've checked with [docs](https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/schema.html#generic-items) and [source code](https://github.com/ckeditor/ckeditor5/blob/43271fc42a4c62a364413a59057f73232592366f/packages/ckeditor5-engine/src/model/schema.ts#L2267). It turns out `allowWhere` adds all `allowIn` items from the parent (ignoring disallowed items, we're not using them), so:
- `allowWhere: '$text'` adds `allowIn: '$block'`
- `allowWhere: '$block'` adds `allowIn: [ '$root', '$container' ]`

I see no issues with that setup. 👍